src/scheduler/resv_info.c

```diff
@@ -352,7 +353,12 @@ query_reservations(server_info *sinfo, struct batch_status *resvs)
 						}
 					}
 				}
-				collect_jobs_on_nodes(resresv->resv->resv_nodes, resresv->resv->resv_queue->jobs, j);
+				jobs_in_reservations = resource_resv_filter(resresv->resv->resv_queue->jobs,
+									    count_array((void **)resresv->resv->resv_queue->jobs),
```

Comment:
These count_arrays don't need to have the argument casted now, right?

Reply:
Sure, I'll rebase